### PR TITLE
tests: add test for headers in compute

### PR DESCRIPTION
**Pull request by georgiyekkert**
1) add test for headers
2) make error message more descriptive for https://github.com/googleapis/java-compute/issues/573

### CODE REVIEW

**Comment by georgiyekkert** [comment]
@vam-google please review

**Review by chanseokoh — APPROVED** [review]

**Review by suztomo — APPROVED** [review]

File: google-cloud-compute/src/test/java/com/google/cloud/compute/v1/integration/ITHeadersTest.java [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+    server.createContext("/", new MyHandler());
+    server.setExecutor(null);
+    server.start();
+    String address = server.getAddress().toString().replace("/", "http://");
+    AddressesSettings addressesSettings =
+        AddressesSettings.newBuilder().setEndpoint(address).build();
+    addressesClient = AddressesClient.create(addressesSettings);
+  }
+
+  static class MyHandler implements HttpHandler {
```
Comment (vam-google) [inline google-cloud-compute/src/test/java/com/google/cloud/compute/v1/integration/ITHeadersTest.java:50]: Please give this class a more descriptive name (now it looks  like a name from a tutorial)

**Review by vam-google — APPROVED** [review]
LGTM, but please rename the class

File: google-cloud-compute/src/test/java/com/google/cloud/compute/v1/integration/ITHeadersTest.java (hunk 1 above)
Comment (georgiyekkert) [reply google-cloud-compute/src/test/java/com/google/cloud/compute/v1/integration/ITHeadersTest.java]: done

**Comment by suztomo** [comment]
@vam-google @georgiyekkert Thank you.

**Pull request merged**